jwst/assign_wcs/util.py

```diff
@@ -659,6 +680,21 @@ def create_grism_bbox(input_model,
                     ymin = int(np.min(ystack))
                     ymax = int(np.max(ystack))
 
+                    # The ``72`` criteria is a placeholder fof when there's a decision
```

Comment:
You can get rid of this comment now, since you've already changed `obj.is_star` to be used as a boolean.